index.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
+entries:
+  reana:
+  - apiVersion: v2
+    created: "2020-04-07T08:17:44.858357473Z"
+    description: REANA reproducible research data analysis platform
+    digest: ed183fde3c43562544cebf244ec99c73d0894841688cb951e706f9de76827bfa
+    home: www.reana.io
+    icon: http://www.reana.io/static/img/logo-reana.svg
+    keywords:
+    - research-data
+    - reproducible-analyses
+    - reproducible-workflows
+    - kubernetes
+    - containers
+    - cwl
+    - reusable-science
```

Comment:
Haven't looked at a keyword list (is there one?) to see what people typically use, but some comments:

- `research-data` better be said `research-data-analyses` or even better just `data-analysis`
- `reusable-science` better be said `reproducible-science` or also `reproducibility`
- another popular term is `scientific-workflows` that we could add
- dunno about another popular term "pipelines", perhaps `scientific-pipelines` or `analysis-pipelines`
- please check also what keywords are used by other systems like Pachyderm, Nextflow, RENKU, etc

Reply:
I've picked the keywords from our GitHub repo:
![Screenshot 2020-04-07 at 13 40 11](https://user-images.githubusercontent.com/9035606/78665006-6c62c180-78d5-11ea-996d-7b2c5e501727.png)
I guess that if we want to update them we should do it everywhere.